src/System.IO.Compression/src/System.IO.Compression.builds

```diff
@@ -5,6 +5,9 @@
     <Project Include="System.IO.Compression.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.IO.Compression.csproj" Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
Yeah, we package the desktop facades in the reference package.  Since this is a partial facade we might want to consider actually building a partial facade reference assembly to slim down the ref package.  That's a separate issue though.